src/main/java/com/example/petbuddybackend/controller/CareController.java

```diff
@@ -140,4 +150,29 @@ public CareDTO cancelCareByClient(@PathVariable Long careId,
         return careService.cancelCareByClient(careId, principal.getName(), TimeUtils.getOrSystemDefault(timeZone));
     }
 
+    @GetMapping("/caretaker-cares")
+    @Operation(
+            summary = "Get filtered cares for caretaker",
+            description = "Returns filtered cares for caretaker. " +
+                    "The result is paginated, sorted and filtered by the provided parameters."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Cares fetched successfully"),
+            @ApiResponse(responseCode = "400", description = "Invalid request data"),
+            @ApiResponse(responseCode = "403", description = "Authorized caretaker can only fetch cares"),
+            @ApiResponse(responseCode = "404", description = "When data provided is not found in the system")
+    })
+    public Page<CareDTO> getCaretakerCares(
+            @RoleParameter @AcceptRole(acceptRole = "CARETAKER") @RequestHeader(value = "${header-name.role}") Role acceptRole,
```

Comment:
```suggestion
            @RoleParameter @AcceptRole(acceptRole = Role.CARETAKER) @RequestHeader(value = "${header-name.role}") Role acceptRole,
```

Ja bym to zmodyfikował żeby to role przyjmowało po prostu

Reply:
w sumie